fw/config.c

```diff
@@ -32,6 +33,7 @@ void config_init(config* self) {
     self->reverse_tt = 0;
     self->tt_effect = SPIN;
     self->tt_deadzone = 4;
+    self->disable_button_led = 0;
   }
 
   update = update_config(self);
```

Comment:
You'll need to add a case under `update_config(..)` or else I don't think this will work correctly for current users (`0xFF ^ 1 == 0xFE`).